src/config/config.c

```diff
@@ -18,83 +18,114 @@ long long backends_intra_op_parallelism; //  number of threads used within an
                                          //  individual op for parallelism.
 long long backends_inter_op_parallelism; //  number of threads used for parallelism
                                          //  between independent operations.
-long long model_chunk_size;              // size of chunks used to break up model payloads.
+long long model_chunk_size;              //  size of chunks used to break up model payloads.
+
+long long onnx_max_runtime; //  The maximum time in milliseconds
+                            //  before killing onnx run session.
+
+static int _RAIConfig_LoadTimeParamParse(RedisModuleCtx *ctx, const char *key, const char *val,
+                                         RedisModuleString *rsval) {
+    int ret = REDISMODULE_OK;
+    if (strcasecmp((key), "TF") == 0) {
+        ret = RAI_LoadBackend(ctx, RAI_BACKEND_TENSORFLOW, (val));
+    } else if (strcasecmp((key), "TFLITE") == 0) {
+        ret = RAI_LoadBackend(ctx, RAI_BACKEND_TFLITE, (val));
+    } else if (strcasecmp((key), "TORCH") == 0) {
+        ret = RAI_LoadBackend(ctx, RAI_BACKEND_TORCH, (val));
+    } else if (strcasecmp((key), "ONNX") == 0) {
+        ret = RAI_LoadBackend(ctx, RAI_BACKEND_ONNXRUNTIME, (val));
```

Comment:
what is this loading section here?

Reply:
This is called from `ai.config LOADBACKEND <backend> <backend_path>` command.